devops/openshift/queue-consumers-deploy.yml

```diff
@@ -211,6 +211,23 @@ objects:
                 requests:
                   cpu: ${CPU_REQUEST}
                   memory: ${MEMORY_REQUEST}
+              readinessProbe:
```

Comment:
some items like `periodSeconds: 10`, `timeoutSeconds: 1`, and `failureThreshold: 3`  are set explicitly with the default values. Was it to make it clear somehow? If that is true, should we include others like `successThreshold`?